distribution/lib/Standard/Base/0.0.0-dev/src/Data/Locale.enso

```diff
@@ -421,14 +416,5 @@ type Locale
     ## Compares two locales for equality.
     == : Any -> Boolean
     == self other = case other of
-        Locale_Data other_java_locale -> self.java_locale.equals other_java_locale
+        Locale.Value other_java_locale -> self.java_locale.equals other_java_locale
         _ -> False
-
-## PRIVATE
-
-   Convert a java locale to an Enso locale.
-
-   Arguments:
-   - java: The java locale value.
-from_java : JavaLocale -> Locale
-from_java java = Locale_Data java
```

Comment:
I see that this is removed. But don't you think it can actually be useful? If someone is doing Java interop they may get hold of a Java locale and a centralized method to convert it could be useful. I'd consider re-adding it and changing `PRIVATE` to `ADVANCED`.

IMO better if users use that method than rely on `Locale.Value` which should indeed be PRIVATE and can change in the future. Whereas here we could provide a forwards-compatible conversion.